app/views/org_admin/sections/_index.html.erb

```diff
@@ -4,8 +4,9 @@
     <div class="panel-group" id="sections_accordion" role="tablist">
       <% sections.each do |section| %>
         <div class="panel panel-default">
+          <% href = (section.modifiable? ? edit_org_admin_template_phase_section_path(template_id: template.id, phase_id: phase.id, id: section.id) : org_admin_template_phase_section_path(template_id: template.id, phase_id: phase.id, id: section.id)) %>
           <a
-            href="<%= org_admin_template_phase_section_path(template_id: template.id, phase_id: phase.id, id: section.id) %>"
+            href="<%= href %>"
```

Comment:
The link now behaves differently when a section is modifiable which is fine but I though we want by default to open the section details in show mode and add edit button if the section is modifiable. I understood that from #1502